is_core/rest/resource.py

```diff
@@ -191,43 +191,43 @@ def get(self):
 
 class RESTModelResource(RESTModelCoreMixin, RESTResource, BaseModelResource):
 
-    default_detailed_fields = None
-    default_general_fields = None
-    extra_fields = None
-    guest_fields = None
     form_class = None
     field_labels = None
     abstract = True
     filters = {}
+    default_fields_extension = ('_rest_links',)
 
     def _get_field_labels(self):
         return (
             self.field_labels if self.field_labels is not None else self.core.get_rest_form_field_labels(self.request)
         )
 
-    def get_extra_fields(self, obj=None):
-        return (
-            self.core.get_rest_extra_fields(self.request, obj=obj) if self.extra_fields is None
-            else rfs(self.extra_fields)
-        )
+    def get_fields(self):
+        fields = super(DefaultRESTModelResource, self).get_fields()
```

Comment:
jinak by se mi fields a ty ostatní níže naplnili hodnotou z meta modelu a to nechci to už si vyřeší core